Now Loads Correctly + Fixed All AVGN Video Links + More [see comment]

esc0rtd3w opened the pull request:
Fixed Loading Issue [Tested on Kodi v16.1 Jarvis]

All AVGN video links have been updated and fixed using the jwplatform urls.

Most links in the XML sitemap still need updated, but the script does load normally and display all content from XML.

Text has been updated and corrected in some instances.

cdwertmann commented:
Thank you for the submission. Most of your changes are however just comments and additions of your name. The core issues that could justify a revival of this plugin are still unsolved:

- most videos don't play (including many AVGN clips)
- no new content since October 2014 since the original feed is gone

If you send me a pull request that fixes those and we can put the plugin back into Kodi.

esc0rtd3w commented:
Yeah, I hear ya. I'll have those resolved soon and updated

On Sat, Jun 4, 2016, 9:07 AM Christoph Dwertmann <notifications@github.com>
wrote:

> Closed #1 <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1>
> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1#event-681924443>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQy4FgSmHcFWvpz0N2WVimbOZ2Adctthks5qIXiZgaJpZM4It9Y5>
> .
>

esc0rtd3w commented:
I have updated my shell script to now parse the new videos. I have
successfully been able to export all needed info to XML files for the Kodi
plugin.

I have started the XML from scratch, this way i can use the shell script to
dump and write the proper formatting.

All of the AVGN videos work, but I still have a few issues. For example the
other categories show up empty but info is present in XML. I have tried to
understand how the ID system was working. I am parsing out the "shortlink"
url and grabbing the number for that.

Here is a link to the shell script if you would like to test it:
https://github.com/esc0rtd3w/cinemassacre-video-dumper/blob/master/cinemassacre-video-dumper.sh

I also have all the dump files as text and xml, uploaded to that project
git (i accidentally put them in the plugin folder the first time).

Thanks for motivating me to update my script so i can update yours! :)

This is a fun project. I have noticed quite a few sites that are not
available in Kodi repos as video/audio plugins, I may take on a few other
projects that are related as well.

On Sat, Jun 4, 2016 at 10:12 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:

> Yeah, I hear ya. I'll have those resolved soon and updated
>
> On Sat, Jun 4, 2016, 9:07 AM Christoph Dwertmann <notifications@github.com>
> wrote:
>
>> Closed #1
>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1>.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1#event-681924443>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/AQy4FgSmHcFWvpz0N2WVimbOZ2Adctthks5qIXiZgaJpZM4It9Y5>
>> .
>>
>

esc0rtd3w commented:
All the videos seem to work when using the "All Videos Sorted By Date"
option

On Mon, Jun 6, 2016 at 1:06 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:

> I have updated my shell script to now parse the new videos. I have
> successfully been able to export all needed info to XML files for the Kodi
> plugin.
>
> I have started the XML from scratch, this way i can use the shell script
> to dump and write the proper formatting.
>
> All of the AVGN videos work, but I still have a few issues. For example
> the other categories show up empty but info is present in XML. I have tried
> to understand how the ID system was working. I am parsing out the
> "shortlink" url and grabbing the number for that.
>
> Here is a link to the shell script if you would like to test it:
>
> https://github.com/esc0rtd3w/cinemassacre-video-dumper/blob/master/cinemassacre-video-dumper.sh
>
> I also have all the dump files as text and xml, uploaded to that project
> git (i accidentally put them in the plugin folder the first time).
>
> Thanks for motivating me to update my script so i can update yours! :)
>
> This is a fun project. I have noticed quite a few sites that are not
> available in Kodi repos as video/audio plugins, I may take on a few other
> projects that are related as well.
>
>
> On Sat, Jun 4, 2016 at 10:12 AM Jason Borowski <esc0rtd3w@gmail.com>
> wrote:
>
>> Yeah, I hear ya. I'll have those resolved soon and updated
>>
>> On Sat, Jun 4, 2016, 9:07 AM Christoph Dwertmann <
>> notifications@github.com> wrote:
>>
>>> Closed #1
>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1>.
>>>
>>> —
>>> You are receiving this because you authored the thread.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1#event-681924443>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe/AQy4FgSmHcFWvpz0N2WVimbOZ2Adctthks5qIXiZgaJpZM4It9Y5>
>>> .
>>>
>>

esc0rtd3w commented:
I am dumping the rest of the links from cinemassacre website for XML. Any
help with the IDs would be helpful. Thanks

On Mon, Jun 6, 2016 at 1:40 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:

> All the videos seem to work when using the "All Videos Sorted By Date"
> option
>
> On Mon, Jun 6, 2016 at 1:06 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:
>
>> I have updated my shell script to now parse the new videos. I have
>> successfully been able to export all needed info to XML files for the Kodi
>> plugin.
>>
>> I have started the XML from scratch, this way i can use the shell script
>> to dump and write the proper formatting.
>>
>> All of the AVGN videos work, but I still have a few issues. For example
>> the other categories show up empty but info is present in XML. I have tried
>> to understand how the ID system was working. I am parsing out the
>> "shortlink" url and grabbing the number for that.
>>
>> Here is a link to the shell script if you would like to test it:
>>
>> https://github.com/esc0rtd3w/cinemassacre-video-dumper/blob/master/cinemassacre-video-dumper.sh
>>
>> I also have all the dump files as text and xml, uploaded to that project
>> git (i accidentally put them in the plugin folder the first time).
>>
>> Thanks for motivating me to update my script so i can update yours! :)
>>
>> This is a fun project. I have noticed quite a few sites that are not
>> available in Kodi repos as video/audio plugins, I may take on a few other
>> projects that are related as well.
>>
>>
>> On Sat, Jun 4, 2016 at 10:12 AM Jason Borowski <esc0rtd3w@gmail.com>
>> wrote:
>>
>>> Yeah, I hear ya. I'll have those resolved soon and updated
>>>
>>> On Sat, Jun 4, 2016, 9:07 AM Christoph Dwertmann <
>>> notifications@github.com> wrote:
>>>
>>>> Closed #1
>>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1>.
>>>>
>>>> —
>>>> You are receiving this because you authored the thread.
>>>> Reply to this email directly, view it on GitHub
>>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1#event-681924443>,
>>>> or mute the thread
>>>> <https://github.com/notifications/unsubscribe/AQy4FgSmHcFWvpz0N2WVimbOZ2Adctthks5qIXiZgaJpZM4It9Y5>
>>>> .
>>>>
>>>

esc0rtd3w commented:
I fixed the XML. All videos show up properly now. I made sure each one had
2 categories. having 2 or more, i am guessing ultimately fixed the issue.

On Mon, Jun 6, 2016 at 1:48 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:

> I am dumping the rest of the links from cinemassacre website for XML. Any
> help with the IDs would be helpful. Thanks
>
> On Mon, Jun 6, 2016 at 1:40 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:
>
>> All the videos seem to work when using the "All Videos Sorted By Date"
>> option
>>
>> On Mon, Jun 6, 2016 at 1:06 AM Jason Borowski <esc0rtd3w@gmail.com>
>> wrote:
>>
>>> I have updated my shell script to now parse the new videos. I have
>>> successfully been able to export all needed info to XML files for the Kodi
>>> plugin.
>>>
>>> I have started the XML from scratch, this way i can use the shell script
>>> to dump and write the proper formatting.
>>>
>>> All of the AVGN videos work, but I still have a few issues. For example
>>> the other categories show up empty but info is present in XML. I have tried
>>> to understand how the ID system was working. I am parsing out the
>>> "shortlink" url and grabbing the number for that.
>>>
>>> Here is a link to the shell script if you would like to test it:
>>>
>>> https://github.com/esc0rtd3w/cinemassacre-video-dumper/blob/master/cinemassacre-video-dumper.sh
>>>
>>> I also have all the dump files as text and xml, uploaded to that project
>>> git (i accidentally put them in the plugin folder the first time).
>>>
>>> Thanks for motivating me to update my script so i can update yours! :)
>>>
>>> This is a fun project. I have noticed quite a few sites that are not
>>> available in Kodi repos as video/audio plugins, I may take on a few other
>>> projects that are related as well.
>>>
>>>
>>> On Sat, Jun 4, 2016 at 10:12 AM Jason Borowski <esc0rtd3w@gmail.com>
>>> wrote:
>>>
>>>> Yeah, I hear ya. I'll have those resolved soon and updated
>>>>
>>>> On Sat, Jun 4, 2016, 9:07 AM Christoph Dwertmann <
>>>> notifications@github.com> wrote:
>>>>
>>>>> Closed #1
>>>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1>.
>>>>>
>>>>> —
>>>>> You are receiving this because you authored the thread.
>>>>> Reply to this email directly, view it on GitHub
>>>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1#event-681924443>,
>>>>> or mute the thread
>>>>> <https://github.com/notifications/unsubscribe/AQy4FgSmHcFWvpz0N2WVimbOZ2Adctthks5qIXiZgaJpZM4It9Y5>
>>>>> .
>>>>>
>>>>

esc0rtd3w commented:
I just finished updating the rest of the XML. Everything works, the XML
needs cleaned up for aesthetic purposes and some links will need added
manually after being double-checked. Its good to go as far as releasing for
public use.

On Mon, Jun 6, 2016 at 2:32 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:

> I fixed the XML. All videos show up properly now. I made sure each one had
> 2 categories. having 2 or more, i am guessing ultimately fixed the issue.
>
> On Mon, Jun 6, 2016 at 1:48 AM Jason Borowski <esc0rtd3w@gmail.com> wrote:
>
>> I am dumping the rest of the links from cinemassacre website for XML. Any
>> help with the IDs would be helpful. Thanks
>>
>> On Mon, Jun 6, 2016 at 1:40 AM Jason Borowski <esc0rtd3w@gmail.com>
>> wrote:
>>
>>> All the videos seem to work when using the "All Videos Sorted By Date"
>>> option
>>>
>>> On Mon, Jun 6, 2016 at 1:06 AM Jason Borowski <esc0rtd3w@gmail.com>
>>> wrote:
>>>
>>>> I have updated my shell script to now parse the new videos. I have
>>>> successfully been able to export all needed info to XML files for the Kodi
>>>> plugin.
>>>>
>>>> I have started the XML from scratch, this way i can use the shell
>>>> script to dump and write the proper formatting.
>>>>
>>>> All of the AVGN videos work, but I still have a few issues. For example
>>>> the other categories show up empty but info is present in XML. I have tried
>>>> to understand how the ID system was working. I am parsing out the
>>>> "shortlink" url and grabbing the number for that.
>>>>
>>>> Here is a link to the shell script if you would like to test it:
>>>>
>>>> https://github.com/esc0rtd3w/cinemassacre-video-dumper/blob/master/cinemassacre-video-dumper.sh
>>>>
>>>> I also have all the dump files as text and xml, uploaded to that
>>>> project git (i accidentally put them in the plugin folder the first time).
>>>>
>>>> Thanks for motivating me to update my script so i can update yours! :)
>>>>
>>>> This is a fun project. I have noticed quite a few sites that are not
>>>> available in Kodi repos as video/audio plugins, I may take on a few other
>>>> projects that are related as well.
>>>>
>>>>
>>>> On Sat, Jun 4, 2016 at 10:12 AM Jason Borowski <esc0rtd3w@gmail.com>
>>>> wrote:
>>>>
>>>>> Yeah, I hear ya. I'll have those resolved soon and updated
>>>>>
>>>>> On Sat, Jun 4, 2016, 9:07 AM Christoph Dwertmann <
>>>>> notifications@github.com> wrote:
>>>>>
>>>>>> Closed #1
>>>>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1>.
>>>>>>
>>>>>> —
>>>>>> You are receiving this because you authored the thread.
>>>>>> Reply to this email directly, view it on GitHub
>>>>>> <https://github.com/cdwertmann/plugin.video.cinemassacre/pull/1#event-681924443>,
>>>>>> or mute the thread
>>>>>> <https://github.com/notifications/unsubscribe/AQy4FgSmHcFWvpz0N2WVimbOZ2Adctthks5qIXiZgaJpZM4It9Y5>
>>>>>> .
>>>>>>
>>>>>
